### PR TITLE
 FUSETOOLS-2736 - UI Tests - Create a new "long" test for "Fuse Ignite...

### PR DESCRIPTION
**Pull request by tsedmik**
# Pull Request Checklist

After this checklist is all checked or PR provides explanations for possible pass-through, please put the label "Ready for review" on the PR.

## General

- [ ] Did you use the Jira Issue number in the commit comments?
- [ ] Did you set meaningful commit comments on each commit?
- [ ] Did you sign off all commits for this PR? (git commit -s -m "jira issue number - your commit comment")

## Functional

- [ ] Did the CI job report a successful build?
- [ ] Is the non-happy path working, too?

## Maintainability

- [ ] Are all Sonar reported issues fixed or can they be ignored?
- [ ] Is there no duplicated code?
- [ ] Are method-/class-/variable-names meaningful?

## Tests

- [ ] Are there unit-tests?
- [ ] Are there integration tests (or at least a jira to tackle these)?
- [ ] Do we need a new UI test?

## Legal

- [ ] Have you used the correct file header copyright comment?
- [ ] Have you used the correct year in the headers of new files?

### CODE REVIEW

**Comment by tsedmik** [comment]
testPR

**Comment by djelinek** [comment]
testPR

**Comment by tsedmik** [comment]
templates job failed due to https://issues.jboss.org/browse/FUSETOOLS-3026

File: uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/NewFuseIgniteExtWizardTest.java [hunk 1]
```diff
@@ -107,23 +107,22 @@ public void testLocation() {
...
 	 * <p>
-	 * Tests the second page (dependencies) of <i>New Fuse Ignite Extension Project</i> wizard
+	 * Tests the second page of <i>New Fuse Ignite Extension Project</i> wizard
 	 * </p>
 	 * <b>Steps:</b>
 	 * <ol>
 	 * <li>Open <i>New Fuse Ignite Extension Project</i> wizard</li>
 	 * <li>Set a project name</li>
-	 * <li>Try to delete each version and check whether user can proceed to another page (UNACCESSIBLE)</li>
+	 * <li>Try to delete Ifnite version and check whether user can finish the wizard (UNACCESSIBLE)</li>
```
Comment (djelinek) [inline uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/NewFuseIgniteExtWizardTest.java]: ..only typography I suppose.. I'f'nite --> I'g'nite

File: uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/NewFuseIgniteExtWizardTest.java [hunk 2]
```diff
@@ -107,23 +107,22 @@ public void testLocation() {
 		firstPage.toggleUseDefaultWorkspaceLocationGroup(true);
 
 		assertTrue(wizard.isNextEnabled());
-		assertFalse(wizard.isFinishEnabled());
 		assertTrue(wizard.isBackEnabled());
 		assertNotEquals(MessageTypeEnum.ERROR, wizard.getMessageType());
 		assertTrue("There are some errors in Error Log", LogGrapper.getPluginErrors("fuse").size() == 0);
```
Comment (djelinek) [inline uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/NewFuseIgniteExtWizardTest.java]: you can use 'LogChecker.assertNoFuseError();'

**Comment by tsedmik** [comment]
@djelinek I incorporated your comments into PR. Can you please recheck the PR? Thank you ;-)

**Comment by djelinek** [comment]
Review done :-) @tsedmik

**Pull request merged**